flowzone.yml

```diff
@@ -2270,6 +2259,19 @@ jobs:
           echo "${result}" >> $GITHUB_OUTPUT
           echo "${DELIMITER}" >> $GITHUB_OUTPUT
 
+      # if auto_merge is enabled at the repository level, configure PR auto-merge
+      - name: Enable PR auto-merge
+        if: |
+          github.event.pull_request.draft == false &&
+          needs.event_types.outputs.do_draft == 'true' &&
+          inputs.repo_config == true
+        run: |
+          if [[ '${{ inputs.repo_enable_auto_merge }}' =~ true ]]; then
```

Comment:
```suggestion
```
This is a setting that may have been enabled at the repo level manually, outside the Flowzone context, so there's no need to check the input here. They might not be using flowzone for repo config.

We should either check the current setting on the repo itself, or just try to enable auto-merge and handle the failure gracefully.

Reply:
I am using the same input to control both the repo level setting and per-PR setting. If we remove this conditional, auto-merge will be enabled always, unless required_approving_review_count is zero. Do we want this?

Reply:
I think if the repo allows it we should be able to toggle it on assuming required reviewers is more than one. So yes. It certainly makes more sense than re-using an optional input meant for applying repo settings on finalize.

Alternately, we could make a new input to control whether to try to enable the auto-merge toggle on the PR (and note that it will fail gracefully if not already allowed in the repo settings).

But if auto-merge is allowed on the repo, and the tests have passed, and at least one person has reviewed the PR, Bulldozer would have merged it anyway so why shouldn't Flowzone?

Reply:
> But if auto-merge is allowed on the repo, and the tests have passed, and at least one person has reviewed the PR, Bulldozer would have merged it anyway so why shouldn't Flowzone?

That's a fair point.